torch/csrc/Module.cpp

```diff
@@ -472,6 +472,20 @@ PyObject *THPModule_deterministic(PyObject *_unused, PyObject *noargs)
   else Py_RETURN_FALSE;
 }
 
+PyObject *THPModule_setWarnOnlyOnce(PyObject *_unused, PyObject *arg)
+{
+  THPUtils_assert(PyBool_Check(arg), "setWarnOnlyOnce expects a bool, "
+          "but got %s", THPUtils_typename(arg));
+  c10::Warning::set_warn_once(arg == Py_True);
+  Py_RETURN_NONE;
+}
+
+PyObject *THPModule_warnOnlyOnce(PyObject *_unused, PyObject *noargs)
+{
+  if (c10::Warning::get_warn_once()) Py_RETURN_TRUE;
```

Comment:
The internal linter will want this to use Egyptian-style curly braces:

```
if ... {
  Py_RETURN_TRUE;
}
```

Reply:
OK. Also modified a few more in the region of this one that used the same pattern.